frontend/src/bundles/friends/components/tab-content/tab-content.tsx

```diff
@@ -34,7 +34,7 @@ const TabContent = ({
                     <FriendCard
                         key={user.userId}
                         user={user}
-                        isFollowed={isFollowed}
+                        isFollowed={isFollowed || !!user.isFollowing}
```

Comment:
use nullish operator

Reply:
@jhladkov 
I can't use the nullish operator here
Because isFollowed is a boolean